docs/reference/run.md

```diff
@@ -1308,6 +1308,10 @@ or two examples of how to pass more parameters to that ENTRYPOINT:
 > **Note**: Passing `--entrypoint` will clear out any default command set on the
 > image (i.e. any `CMD` instruction in the Dockerfile used to build it).
 
+> **Note**: Passing `--entrypoint=""` (empty string) will reset the entry
+> point to system default (i.e., the entry point used by docker when there
+> is no `ENTRYPOINT` instruction in the Dockerfile).
```

Comment:
Wondering; I'm not at my computer now, but does resetting the entrypoint also reset the `CMD` (the same as setting a custom entrypoint?)

If so, I think we should have a test case for that